bin/pycbc_live

```diff
@@ -1242,6 +1254,34 @@ with ctx:
                         if len(results[ifo][key]):
                             results[ifo][key] = results[ifo][key][idx]
 
+            # Calculate and add the psd variation for the results
+            if args.psd_variation:
+                filt = {ifo: None for ifo in ifos}
+
+                for ifo in results:
+                    logging.info(f"Calculating PSD Variation Statistic for {ifo}")
+
+                    if filt[ifo] is None:
```

Comment:
Similarly here, how can this ever not be None, 6 lines earlier it was reset to None, regardless of what happens further down at line 1282. Should `filt = {ifo: None for ifo in ifos}` also move outside of the main `while` loop? If so, it needs a more informative variable name.

Reply:
I have moved filt (and renamed) to outside of the main loop. I don't want this being reset at all once the search has commenced, only remade as the psd gets recalculated. I've also changed the logic on the remaking as I think it was wrong. Just testing now